lighthouse-core/gather/computed/metrics/lantern-first-cpu-idle.js

```diff
@@ -56,8 +52,19 @@ class LanternFirstCPUIdle extends LanternInteractive {
       longTasks.push({start: timing.startTime, end: timing.endTime});
     }
 
+    // Require here to resolve circular dependency.
+    const FirstCPUIdle = require('./first-cpu-idle');
```

Comment:
I found the new structure for computeSimulatedMetrics 👍

Reply:
> I found the new structure for computeSimulatedMetrics 👍

yeah, this seemed fine for now since it's just two places, but we can always refactor into helper functions for things like `findQuietWindow`